ops/README.md

```diff
@@ -28,6 +28,14 @@ docker-compose \
     up --build --detach
 ```
 
+Do the same to run a verifier along side the regular stack.
+```
+docker-compose \
+    -f docker-compose.yml \
+    -f verifier.yml \
```

Comment:
Nit: follow the style of the `docker-compose-` prefix to keep things a more unified style, see the metrics example above